docs/deployment/prerequisites.md

```diff
@@ -0,0 +1,75 @@
+---
+title: UDS Prerequisites
+type: docs
+weight: 4
+---
+
+## UDS installation prerequisites
+
+`UDS Core` could run in any [Kuberentes](https://kubernetes.io/) setup, but sometimes customizations are needed based on environemnts. This is an attempt to document and link to relevant information to aid in setting up your [Kuberentes](https://kubernetes.io/) environment to ensure a successful `UDS Core` installation.  
+
+### RHEL
+---
+#### *ISTIO related changes*
+Solution is to create file `/etc/modules-load.d/istio-iptables.conf` with this content:
+
+```bash
+# These modules need to be loaded on boot so that Istio (as required by
+# UDS Core) runs properly.
+#
+# See also: https://github.com/istio/istio/issues/23009
+
+br_netfilter
+nf_nat
+xt_REDIRECT
+xt_owner
+iptable_nat
+iptable_mangle
+iptable_filter
+```
+
+```bash
+sudo systemctl stop firewalld
+sudo systemctl disable firewalld
+```
```

Comment:
I think for these Istio pieces as well, they are more broadly applicable than just RHEL. I think the doc you linked in the Istio section below is good, maybe we could also add [this code snippet](https://github.com/defenseunicorns/uds-rke2-image-builder/blob/main/packer/scripts/os-prep.sh#L28-L33) in that section and delete this RHEL specific section.

(also noting that the firewalld issue is captured under the RKE2 bullets)